### ISSUE TITLE
No indication of how to save, or that saving is automatic, or to prevent saving.

### ISSUE THREAD

**Issue by lf94** [KittyCAD/modeling-app#1929]
ZMA does not tell the user saving is automatic.

There are users who may also make changes but not wish to save those changes. What can they do?

What about users that want to save the project under another name?

I was genuinely confused if closing ZMA was going make me lose my KCL.

**Comment by jessfraz** [comment]
this is a good point because I often compare the behavior of our app to vscode, there are lots of similarities:

- local files
- some extensions in vscode call out to a cloud but all files are still local

but a big difference would be in vscode you inheritly hit save, in that regard it is pretty unintuitive, maybe something for the onboarding?

adding @franknoirot

**Comment by franknoirot** [comment]
Yeah this is one of the points along which I think our app borrows from Figma not VS Code. I propose two things:
1. Adding early in the onboarding that saving is automatic (we should also mention that they are saved locally)
2. Add a little toast like Figma used to show when users hit <kbd>⌘S</kbd> or <kbd>Ctrl+S</kbd> that says "All your changes are auto-saved"

Here are some notes from Figma on their autosave feature: https://www.figma.com/blog/behind-the-feature-autosave/
